projects/js-packages/components/components/pricing-table/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { Fragment, createContext, useContext, Children, cloneElement } from '@wordpress/element';
```

Comment:
I think we should rely on React here instead of `@wordpress/element`

Reply:
Good call, I'll change that. 👍🏼